.github/workflows/test_frameworks.yml

```diff
@@ -230,15 +238,18 @@ jobs:
         env:
           # Disabled distributed tracing since there are a lot of tests that assert on headers
           DD_HTTPX_DISTRIBUTED_TRACING: "false"
+          # Debugger exploration testing does not work in CI
+          # PYTHONPATH: ../ddtrace/tests/debugging/exploration/
         # test_pool_timeout raises RuntimeError: The connection pool was closed while 1 HTTP requests/responses were still in-flight
-        run: pytest -k 'not test_pool_timeout'
+        run: pytest -k 'not test_pool_timeout and not test_deleting_unclosed_async_client_causes_warning'
```

Comment:
is this change required still if we're not doing the exploration tests for this job

Reply:
ah right! I guess this can be reverted.